discord/bot.py

```diff
@@ -23,6 +23,12 @@ async def good_button(
         await interaction.response.send_message(
             content="This is an edited button response!"
         )
+        filename = f"discord/private_chats/{interaction.user.id}.txt"
```

Comment:
This implementation is OS-specific
See: https://docs.python.org/3/library/pathlib.html